docs/source/app-dev/bindings-java/quickstart.rst

```diff
@@ -122,28 +132,33 @@ In this section, you will run the quickstart application and get introduced to t
 
    .. _quickstart-sandbox:
 
-#. To run the :doc:`sandbox </tools/sandbox>` (a lightweight local version of the ledger), run ``daml sandbox --dar .daml/dist/quickstart-0.0.1.dar``
+#. Run ``daml sandbox --port 6865``
 
-   The output should look like this:
+#. In a separate terminal run the following:
 
-   .. code-block:: none
+- ``daml ledger upload-dar --host localhost --port 6865 .daml/dist/quickstart-0.0.1.dar``
 
-      Starting Canton sandbox.
-      Listening at port 6865
-      Uploading .daml/dist/quickstart-0.0.1.dar to localhost:6865
-      DAR upload succeeded.
-      Canton sandbox is ready.
+- ``daml script --ledger-host localhost --ledger-port 6865 --dar .daml/dist/quickstart-0.0.1.dar --script-name Main:initialize --output-file output.json``
 
+- ``daml navigator server localhost 6865 --port 7500``
 
-   The sandbox is now running, and you can access its :doc:`ledger API </app-dev/ledger-api>` on port ``6865``.
+#. In a third terminal, run ``daml codegen java && mvn compile exec:java@run-quickstart -Dparty=$(cat output.json | sed 's/\[\"//' | sed 's/".*//')``
```

Comment:
The maven steps are way down at the moment. Can we move this part down as well? Having it be in here while the rest of the Java bindings instructions are all the way down seems somewhat confusing.

Reply:
To make sure I have this correct, you mean down in the Integrate With the Ledger section?

Reply:
as discussed yes, add the `-Dparty` stuff to the two `mvn exec` calls in `integrate with the ledger`.